### DISCUSSION TITLE
How to compose docker images after source code change

### DISCUSSION THREAD

**Discussion by CodeSkills** [postalserver/postal#2339] [Help with using Postal] [answered]
Hey there,

I need to disable IPv6 as in the https://github.com/postalserver/postal/issues/435 but has no experience with composing docker images. I tried cloning the repo, and doing docker compose, but it requires some POSTAL_IMAGE variable.

How can I create that?

**Comment by willpower232** [comment, marked as answer]
Judging by the makefile, you can just build the image with `make docker-build` I think but you might find it easier to overwrite the file with the volumes in the docker-compose.yml file that runs the whole thing.

**Reply by CodeSkills** [reply]
I appreciate your help. I tried to find the file but I only managed to access the configs not the source files within the docker containers, how can I access the source within the running containers?

**Reply by CodeSkills** [reply]
To make it easier, let's say, can I somehow override the whole Postal code within the container to be on the host, so I can make any changes to it and just restart the containers with that being as a volume when making changes? I am trying to find where is the code stored within the container to try it myself but am not succeeding

**Reply by willpower232** [reply]
You should be able to download the file https://github.com/postalserver/postal/blob/master/lib/postal/smtp_server/server.rb and then map it to `/opt/postal/app/lib/postal/smtp_server/server.rb`

**Reply by CodeSkills** [reply]
Thanks! It still did not solve my issue, but at least taught me how to do this :-) Will create a new discussion for the issue I am facing.

Thanks again!
